vc-wallet-web/Makefile

```diff
@@ -0,0 +1,131 @@
+#
```

Comment:
VCS adds components for issuer, verifier and holder https://github.com/trustbloc/k8s/tree/main/vcs/kustomize/vcs/overlays - we can follow similar structure for this rather than high level folder. cc: @alialkhalidi

Reply:
not sure I follow, is vc-wallet-web another mode of vcs (the others being holder, issuer, verified, combined)? can it be? if so, then yes, it makes sense to use the same vcs base for it.

Otherwise, I'm not sure where this component fits within the overall [system](https://github.com/trustbloc/sandbox/blob/main/docs/components/vcs_components.md)

Reply:
@alialkhalidi vc-wallet-web is another instance of [wallet-web](https://github.com/trustbloc/k8s/tree/main/wallet-web)

Reply:
> @alialkhalidi vc-wallet-web is another instance of [wallet-web](https://github.com/trustbloc/k8s/tree/main/wallet-web)

Ah, I see, yes then it make sense NOT to create a new top-level component, rather use a similar technique in wallet-web as vcs.

Reply:
Ok i will fix it